examples/run_largecyc_generic_echo.py

```diff
@@ -136,7 +134,8 @@
     [len(ph1), len(ph_overall), -1],
 )
 data.setaxis("ph1", ph1 / 4).setaxis("ph_overall", ph_overall / 4)
-data.set_prop("postproc_type", "spincore_generalproc_v1")
+data.set_units("t2", "s")
+data.set_prop("postproc_type", "spincore_largecyc_generic_echo_v1")
```

Comment:
increment post processing version so that we aren't multiplying by nScans anymore and we currently don't have any function to process this guy so v1